src/modules/profile/ProfileScreen.tsx

```diff
@@ -1,37 +1,179 @@
+import { Pressable, StyleSheet, Text, View, Platform, Image, ImageBackground, ScrollView } from 'react-native';
 
 import { Routes } from '../../constants/Routes';
+import { Colors } from '../../constants/Colors';
 import { RootStackScreenProps } from '../../constants/types';
+import { Input, StyledLine, Switch } from '../../components';
+
+const DEFAULT_URI_AVATAR = '../../../assets/images/avatar.png';
+const DEFAULT_URI_BACKGROUD = '../../../assets/images/backgr.png';
+
+const getPermission = async () => {
+  if (Platform.OS !== 'web') {
+    const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+    if (status !== 'granted') {
+      alert('Sorry, we need camera roll permissions to make this work!');
+      return false;
+    } else {
+      return true;
+    }
+  }
+  return true;
+}
+
+const onChange = (value: string, callBack: Function) => {
+  callBack(() => value)
+}
+
+export const ProfileScreen = ({ navigation }: RootStackScreenProps<Routes.profile>) => {
+  const [image, setImage] = React.useState('');
+  const [bcground, setBcground] = React.useState('');
+  const [name, setName] = React.useState('Anastasiia');
+  const [lightTheme, setLightTheme] = React.useState(true);
+  const [email, setEmail] = React.useState('email@gmail.com');
+  const [nickName, setNickName] = React.useState('My NickName');
+  const [pushNotification, setPushNotification] = React.useState(false);;
+
+  const pickImage = async () => {
+    let isGranted = await getPermission();
+    if (isGranted) {
+      let result = await ImagePicker.launchImageLibraryAsync({
+        mediaTypes: ImagePicker.MediaTypeOptions.All,
+        allowsEditing: true,
+        aspect: [4, 3],
+        quality: 1,
+      });
+
+      if (!result.cancelled) {
+        setImage(result.uri);
+      }
+    }
+  };
+
+  React.useEffect(() => {
+    (async () => setImage(Asset.fromModule(require(DEFAULT_URI_AVATAR)).uri))();
+    (async () => setBcground(Asset.fromModule(require(DEFAULT_URI_BACKGROUD)).uri))();
+  }, [])
 
-export function ProfileScreen({ navigation }: RootStackScreenProps<Routes.profile>) {
   return (
+    <ScrollView style={styles.scroll}>
     <View style={styles.container}>
-      <Pressable onPress={() => navigation.replace(Routes.login)} style={styles.link}>
-        <Text style={styles.linkText}>Profile screen!</Text>
-      </Pressable>
+      <ImageBackground style={styles.imageContainer} source={{ uri: bcground }}>
+        <Pressable onPress={pickImage} style={styles.imageWrape}>
+          <Image
+            style={styles.image}
+            source={{ uri: image }}
+          />
+        </Pressable>
+
+      </ImageBackground >
+
+      <View style={styles.infoContainer}>
+
+        <Text style={styles.title}> Profile </Text>
+        <Input 
+          icon={undefined} 
+          value={nickName} 
+          placeholder={'enter your nick'} 
+          onChange={(value: string) => onChange(value, setNickName)}
```

Comment:
What is the point of using `onChange` here? Can't we just use `setNickName` directly?
```suggestion
          onChange={setNickName}
```
?

Reply:
))) it's my habit from unit-testing ))  will be done!